resources/qml/Dialogs/RenameDialog.qml

```diff
@@ -55,32 +59,58 @@ UM.Dialog
             id: nameField
             width: parent.width
             text: base.object
+            placeholderText: base.objectPlaceholder
+            placeholderTextColor: UM.Theme.getColor("text_field_text_disabled")
             maximumLength: 40
             selectByMouse: true
             onTextChanged: base.textChanged(text)
         }
 
+        // spacer
+        Rectangle
+        {
+            color: base.backgroundColor
+            height: UM.Theme.getSize("wide_margin").height
+            width: height
+        }
```

Comment:
```suggestion
        Item
        {
            height: UM.Theme.getSize("wide_margin").height
            width: height
        }
```

Sometimes I one line these spacers. Not really sure it is best practice though

```suggestion
        Item { height: UM.Theme.getSize("wide_margin").height; width: height } // Spacer
```

Reply:
Maybe we should just create a number of default spacers in UM namespace? 

It would make sense to have a narrow & wide spacer that we can re-use. It also makes it clearer that it's really just a spacer.

Reply:
I agree, but it would be good to do that another time, not on Friday evening ;-)